### DISCUSSION TITLE
Bypassing the "Cannot access contents of url "chrome-error://chromewebdata/"." Error

### DISCUSSION THREAD

**Discussion by doroness** [AutomaApp/automa#362] [Q&A]
Hi
I'm using Automa to check URLs. Some don't exist anymore (e.g. http://thisdontexcists.com/). I'm trying to run some JS code that uses the `{{activeTabUrl}}`. IF  the URL produces the "DNS_PROBE_FINISHED_NXDOMAIN" - the following error is logged.

_Cannot access contents of URL "chrome-error://chromewebdata/". Extension manifest must request permission to access this host._

Is there a way to avoid this? i.e if-then statement or somehow provide the needed permission? (so the flow won't stop).

<img width="1315" alt="image" src="https://user-images.githubusercontent.com/53006624/154632446-8546fad7-ab9f-4f4b-af8a-9e9039bd713d.png">

**Comment by Kholid060** [comment]
It's currently not possible to bypass that, but I had added a [HTTP request block](1a7fe893b40c415ea321d888d3f94121a2b5f877) with a fallback, so you can use that to check if the web exists or not

**Reply by doroness** [reply]
Hi. Thank you. That sounds good. I'm using the chrome extension(version 0.17.4), and I don't see that block. How can I use it?
